tests/settings.py

```diff
@@ -4,16 +4,14 @@
         "tests.utils.app_2",
         "djangocms_versioning",
         "djangocms_version_locking",
+
+        # the following 4 apps are related
+        "aldryn_forms",
         "filer",
         "easy_thumbnails",
-        "absolute",
-        "aldryn_forms",
         "captcha",
-        "emailit",
+
         "djangocms_text_ckeditor",
-        "djangocms_versioning.test_utils.blogpost",
```

Comment:
Should these versioning packages be removed?

Reply:
Yes they are not necessary. Tests run fine without them, and these are really for internal use for djangocms-versioning tests.